vshard/storage/init.lua

```diff
@@ -2873,13 +2925,19 @@ local function rebalancer_apply_routes_f(routes)
         local f = workers[i]
         local ok, res = f:join()
         if not ok then
-            log.error('Rebalancer worker %d threw an exception: %s', i, res)
+            local err_msg = string.format(
+                'Rebalancer worker %d threw an exception: %s', i, res)
+            service:set_status_error(err_msg)
+            log.error(err_msg)
         end
     end
-    if not next(dispenser.errors) then
+    if not dispenser.error then
```

Comment:
This should have been fixed in the original commit.

Reply:
Here we go)